insecure/orchestrator/orchestratorNetwork.go

```diff
@@ -0,0 +1,250 @@
+package orchestrator
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/insecure"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/component"
+	"github.com/onflow/flow-go/module/irrecoverable"
+	"github.com/onflow/flow-go/network"
+	"github.com/onflow/flow-go/network/channels"
+	"github.com/onflow/flow-go/utils/logging"
+)
+
+// OrchestratorNetwork implements a middleware for mounting an attack orchestrator and empowering it to communicate with the corrupt nodes.
+type OrchestratorNetwork struct {
```

Comment:
remove packge name from struct name,` orchestrator.Network` is better than `orchestrator.OrchestratorNetwork`

Reply:
@gomisha please note that in `Golang` this is considered an anti-pattern to start a component name with its package name. Hence, `orchestrator.OrchestratorNetwork` is a naming anti-pattern and `orchestrator.Network` is recommended instead. 

From [https://go.dev/blog/package-names](https://go.dev/blog/package-names):
> Avoid repetition. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the `http` package is called `Server`, not `HTTPServer`. Client code refers to this type as `http.Server`, so there is no ambiguity.

Reply:
renamed orchestrator.OrchestratorNetwork to orchestrator.Network, updated main and test file names

https://github.com/onflow/flow-go/pull/2936/commits/c3dbd41a913eea33933ecc9479d4b301a3a427e9